microsoft-edge/devtools-guide-chromium/whats-new/2023/05/devtools-113.md

```diff
@@ -1,68 +1,123 @@
+
+![Navigating the Issues tool with the keyboard no longer causes the Close button to disappear](./devtools-113-images/issues-tool-regenerate-infobar-close-button.png)
+
+See also:
+* [Filter issues](../../../issues/index.md##filter-issues) in _Find and fix problems using the Issues tool_
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
 
 <!-- ====================================================================== -->
-## Entry 3
+## In Focus Mode, hovering over a tool no longer makes the title of each tool disappear in the Activity Bar
+
+<!-- Subtitle: With the Activity Bar in the horizontal orientation, you can now see the title of each tool and hover over the tool icon to see a tooltip. -->
+<!-- Reviewer: Yanling Wang -->
 
-<!-- Subtitle: . -->
+In previous versions of Microsoft Edge, in Focus Mode, hovering over a tool in the Activity Bar would cause all the titles of the tools in the Activity Bar to disappear.  In Microsoft Edge 113, this issue has been fixed.  With the Activity Bar at the top, you can hover over tool icons to see the name of the tool in a tooltip and you can still see all of the titles of the tools in the toolbar.
 
-* [Alt text 3](./devtools-113-images/template.png)
+![Hovering over a tool icon in the Activity Bar produces a tooltip with the name of the tool](./devtools-113-images/focus-mode-tooltip.png)
 
-<!-- 
 See also:
-* []()
--->
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
 
+## Context menus in the DevTools better support themes from VS Code
 
-<!-- ====================================================================== -->
-## Entry 4
+<!-- Subtitle: Customize DevTools to match VS Code with themes like Solarized Light, Kimbie Dark, Monokai, or Tomorrow Night Blue. -->
+<!-- Reviewer: Irene Cho -->
 
-<!-- Subtitle: . -->
+In previous versions of Microsoft Edge, applying themes from VS Code like Solarized Light or Kimbie Dark caused low contrast issues when hovering over an item in a context menu.  In Microsoft Edge 113, this issue has been fixed.  Context menus now display better, with sufficient color contrast, when using themes in DevTools.
 
-* [Alt text 4](./devtools-113-images/template.png)
+![Sufficient color contrast in a context menu in DevTools with the Solarized Light theme](./devtools-113-images/themes-contrast-in-context-menu.png)
 
-<!-- 
 See also:
-* []()
--->
+* [Apply a color theme to DevTools](../../../customize/theme.md)
+* [Test text-color contrast using the Color Picker](../../../accessibility/color-picker.md)
+* [Check text-color contrast in the default state using the Inspect tool](../../../accessibility/test-inspect-text-contrast.md)
+
+## The Node types filter in the Memory tool now only displays in the Summary view of a heap snapshot
+
```

Comment:
@zoherghadyali - what's the Welcome Subtitle string?
<!-- Subtitle: todo. -->

Reply:
```
<!-- Subtitle: The Containment and Statistics views for heap snapshots no longer display the Node Types filter, since the filter doesn't apply to those views. -->
```